consensus/scp/src/quorum_set.rs

```diff
@@ -39,18 +29,68 @@ pub enum QuorumSetMember<ID: GenericNodeId> {
     InnerSet(QuorumSet<ID>),
 }
 
-impl<
-        ID: GenericNodeId
-            + Clone
-            + Debug
-            + Display
-            + Serialize
-            + DeserializeOwned
-            + Eq
-            + PartialEq
-            + Hash,
-    > QuorumSet<ID>
-{
+impl<ID: GenericNodeId> PartialEq for QuorumSetMember<ID> {
```

Comment:
How is this different than the default `Eq` implementation?

Reply:
I think I just got overzealous with implementing this. Everything seems to work fine with leaving these derived for `QuorumSetMember`. Thanks for the catch.